tensorflow_addons/layers/python/normalizations.py

```diff
@@ -0,0 +1,363 @@
+# Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# =============================================================================
+
+# Orginal implementation from keras_contrib/layer/normalization
+
+from tensorflow.keras import backend as K
+from tensorflow.keras import constraints
+from tensorflow.keras import initializers
+from tensorflow.keras import regularizers
+from tensorflow.keras.layers import InputSpec
+from tensorflow.keras.layers import Layer
+from tensorflow.python.ops import nn
+
+
+class GroupNormalization(Layer):
+    """Group normalization layer.
+
+    Group Normalization divides the channels into groups and computes
+    within each group the mean and variance for normalization.
+    Group Normalization's computation is independent
+    of batch sizes, and its accuracy is stable in a wide range of batch sizes.
```

Comment:
I'm not very comfortable seeing "too broad" statements like this. The minimum pre-condition needed on this statement would be:
__Empirically__, its accuracy is __more__ stable __than batch norm__ in a wide range of __small__ batch sizes, __if learning rate is adjusted linearly__ with batch sizes.

I don't know if documentation is actually a suitable place for writing down such empirical findings.
disclaimer: author of the paper

Reply:
@ppwwyyxx thank you for your feedback. 
I hope it is okay for you that I just added this sentence to the documentation.